Mathlib/FieldTheory/Galois/Notation.lean

```diff
@@ -0,0 +1,41 @@
+/-
+Copyright (c) 2025 Andrew Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Andrew Yang
+-/
+import Mathlib.Algebra.Algebra.Equiv
+import Lean.PrettyPrinter.Delaborator.Basic
+
+/-!
+# Notation for galois group
+
+The galois group `Gal(L/K)` is implemented via `L ≃ₐ[K] L` in mathlib.
+We provide such a notation in this file.
+
+Although this notation works for all automorphism groups of algebras, we should only use this
+notation when `L/K` is an extension of fields.
+-/
+
+section Notation
+
+/--
+Notation for `Gal(L/K) := L ≃ₐ[K] L`.
+
+Although this notation works for all automorphism groups of algebras, we should only use this
+notation when `L/K` is an extension of fields.
+-/
+notation "Gal(" L:100 "/" K ")" => L ≃ₐ[K] L
```

Comment:
We should probably document in a comment that this is chosen to be above the precedence for division, e.g.
```
/- We use a precedence of 100 here to avoid `L` capturing `"/"` as the infix notation for division, 
which has precedence 70. -/
```
(By the way, any reason for 100 specifically, besides "safely above 70"?)

Reply:
Yeah there is no particular reason.